apps/passport-server/src/services/generic-issuance/pipelines/PODPipeline/PODPipeline.ts

```diff
@@ -341,7 +341,7 @@ export class PODPipeline implements BasePipeline {
           );
 
           // TODO(artwyman): Figure out how to phase out this use of the deprecated format.
-          const id = uuidv5(serializePODEntries(entries), this.id);
+          const id = uuidv5(JSON.stringify(podEntriesToJSON(entries)), this.id);
```

Comment:
Is there any possibility that the implementation of `podEntriesToJSON` might change in the future, such that the keys and values would be identical but the keys might be inserted in a different order? If so we could use `fast-stable-stringify` here.

Reply:
In both of these codepaths, the entries are being sorted by PODContent.fromEntries().asEntries() so they should be in canonical order.  podEntriesToJSON doesn't change that, and will pass through entries in the same order.  All the values have at most one entry, so there shouldn't be way for the order to differ.  And JSONification always picks the same value format for any given value (particularly numbers which differ based on size).

That being said, stability isn't an explicit design goal of the JSON format, and the format may change in a backward-compatible way.  That's why this isn't a great approach to generating a stable hash.